Teste falhando em test_view_customer_add

[test_view_customer_add](https://github.com/rg3915/django-experience/blob/master/djexperience/bookstore/tests/test_view_customer_add.py)
```
FAIL: test_post (djexperience.bookstore.tests.test_view_customer_add.CustomerAddPostValid)
Valid POST should redirect to /customer_detail/1/
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rg3915/gh/my/django-experience/djexperience/bookstore/tests/test_view_customer_add.py", line 56, in test_post
    self.assertRedirects(self.resp, r('bookstore:customer_detail', 1))
  File "/home/rg3915/gh/my/django-experience/.venv/lib/python3.5/site-packages/django/test/testcases.py", line 315, in assertRedirects
    (response.status_code, status_code))
AssertionError: 200 != 302 : Response didn't redirect as expected: Response code was 200 (expected 302)

======================================================================
FAIL: test_save_customer (djexperience.bookstore.tests.test_view_customer_add.CustomerAddPostValid)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rg3915/gh/my/django-experience/djexperience/bookstore/tests/test_view_customer_add.py", line 59, in test_save_customer
    self.assertTrue(Customer.objects.exists())
AssertionError: False is not true

```

--- a/djexperience/bookstore/tests/test_view_customer_add.py
+++ b/djexperience/bookstore/tests/test_view_customer_add.py
@@ -1,5 +1,4 @@
 from django.test import TestCase
-from django.utils import timezone
 from django.shortcuts import resolve_url as r
 from djexperience.bookstore.forms import CustomerForm
 from djexperience.bookstore.models import Customer
@@ -47,7 +46,7 @@ class CustomerAddPostValid(TestCase):
             first_name='Adam',
             last_name='Smith',
             email='adam@example.com',
-            birthday=timezone.now()
+            birthday='2006-10-25 14:30:59'
         )
         self.resp = self.client.post(r('bookstore:customer_add'), data)
 

